pkg/storage/config/store.go

```diff
@@ -38,7 +39,7 @@ func (cfg *ChunkStoreConfig) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.L2ChunkCacheHandoff, "store.chunks-cache-l2.handoff", 0, "Chunks will be handed off to the L2 cache after this duration. 0 to disable L2 cache.")
 	f.BoolVar(&cfg.chunkCacheStubs, "store.chunks-cache.cache-stubs", false, "If true, don't write the full chunk to cache, just a stub entry.")
 	cfg.WriteDedupeCacheConfig.RegisterFlagsWithPrefix("store.index-cache-write.", "", f)
-
+	f.DurationVar(&cfg.SkipQueryWritebackOlderThan, "store.skip-query-writeback-older-than", 0, "Chunks fetched from queriers before this duration will not be written to the cache. A value of 0 will write all chunks to the cache")
```

Comment:
nit: wdyt about `SkipCachingChunksOlderThan`?

Reply:
The reason for including `Query` in the name was to distinguish that these were the chunks being written by the querier to the cache. This configuration only applies to the query operation.

If I needed to ingest historical data from 3 months ago into Loki, using `SkipCachingChunksOlderThan` might imply that the ingested data is not written to the cache